### ISSUE TITLE
Disable automatic zoom-out when a new location update is being received (frontend)

### ISSUE THREAD

**Issue by Jotager** [owntracks/frontend#41]
When i zoom in looking at my location history it only stays zoomed in for a few seconds until it zooms back out to the "fit view" option. 
I assume this happens whenever a new update is received since the behavior did not occur when disabling my internet connection
Please correct me if im wrong!

**Comment by linusg** [comment]
You're not wrong - whenever the local location history or list of "last locations" changes, the map is re-fitted to the new data:

https://github.com/owntracks/frontend/blob/6180e52f71f7c8cf9afbe8f342172448b543d06a/src/views/Map.vue#L233-L240

Would you like a config option to disable this behaviour?

**Comment by Jotager** [comment]
Yes id be very thankful if this would be disabled by default / a persistent setting to disable this!
Thanks

**Comment by linusg** [comment]
@Jotager I disabled this behaviour by default in https://github.com/owntracks/frontend/commit/206eb268fa9fdc84f6de88ccc7d0aa8603fbc75e, it can be re-enabled by setting `onLocationChange.fitView` to `true`.

Released as of 2.5.0, Docker images will build shortly in case you use those. As I said, no config change needed in your case - upgrading is enough. I hope that resolves the issue for you, if not let me know!
